### DISCUSSION TITLE
Does avoid_negative_ts = auto NOT select "make non negative" when merging?

### DISCUSSION THREAD

**Discussion by Mrnofish** [mifi/lossless-cut#2031] [Q&A]
When switching mode to "merge cuts", a warning appears next to "avoid_negative_ts" if it's set to auto, recommending to change the setting to "make non negative".

This means if I'm alternating between Separate and Merge I'm constantly changing avoid_negative_ts between auto and make non negative.

Why does auto do not activate "make non negative" for merging if that's the recommended setting?
What does it do instead?

**Comment by mifi** [comment]
`auto` means use ffmpeg's `auto` value. from experience, we've found that `auto` does often not give the best result when merging. that's why that warning was added. but YMMV

**Reply by Mrnofish** [reply]
I see. May I suggest what I believe would be improvements?

1. Change that setting automatically, e.g. when Merge is selected, default to make_non_zero; when the mode is changed again, revert to the previous value, be it auto or whatever the user selected. This would save a ton of clicking.
2. Unify the tooltips for the various avoid_negative_ts values: if I want to read what each value does, without leaving LosslessCut, I have to cycle manually through each option.
3. When make_non_zero is selected, clicking on the warning sign does not open any tooltip, making it impossible to find out what it does from within the program.
4. Make it explicit that avoid_negative_ts is not a LosslessCut option (in hindsight it is obvious but as someone who never uses ffmpeg from the command line, I just didn't connect the dots)

**Reply by mifi** [reply]
created a feature request: #2102
